dspace-api/src/main/java/org/dspace/content/authority/DCInputAuthority.java

```diff
@@ -44,10 +50,10 @@
 public class DCInputAuthority extends SelfNamedPlugin implements ChoiceAuthority {
     private static Logger log = org.apache.logging.log4j.LogManager.getLogger(DCInputAuthority.class);
 
-    private String values[] = null;
-    private String labels[] = null;
+    private Map<String, String[]> values = null;
```

Comment:
Can you add JavaDocs explaining what values and labels are now?
If values is a map, it's not clear what the keys and values of the map are (and similar for labels)

Reply:
done